package.json

```diff
@@ -2,44 +2,76 @@
   "name": "clickhouse-datasource",
   "version": "2.1.1",
   "description": "Clickhouse Datasource",
+  "engines": {
+    "node": ">=16"
+  },
   "scripts": {
-    "spellcheck": "cspell -c cspell.config.json \"**/*.{ts,tsx,js,go,md,mdx,yml,yaml,json,scss,css}\"",
```

Comment:
We should keep the spellcheck script.

Reply:
And `watch` for a better developer experience as it seems to be missing from the new template. I will add them back :+1:

Reply:
@bossinc I've added `cspell` dev dependency and the `spellcheck` script.